### ISSUE TITLE
Web page loses some image layers upon page download event

### ISSUE THREAD

**Issue by Dboingue** [deathau/markdownload#362]
Is that known. I have not checked if that is only after the click on the extension icon, and the base64 images would still be having the disapeared images chunks.  If that is not already in another issue (I searched for image, on open issues), then I can reproduce and share pics of what I mean.  It does not seem to be all the images, on a page. But for example saving from ChatGPT.com, will disappear the GPT response text chunk turn marker.  Not that much of a problem there. But I have encoutered it on other sites, for example on lichess, the drawings get diappeared.  It might be a web techonology layer.  But other extensions can save the hmtl that renders back the visual. 

Is this a markdown limitation, or base64 encoding.  I am just guessing here. I would like to be able to control that kind of effect.
On GPT I then have to reload to get back the images.  But on the chess site I lose the work of drawing I did.  

I thought I would report that. Not really why I use the extension. The sizes of Markdown  files and the ability to parse them with various tools while keeping some human content structure, would make such a better way for me to keep my web traversal and excerpts in my folder, than keeping all the html layers an so. And I don't mean either of the 2 examples I could use to explain if this is not known.
